packages/xo-web/src/xo-app/logs/backup-ng-logs.js

```diff
@@ -185,7 +185,15 @@ const LOG_FILTERS = {
 
 export default [
   addSubscriptions({
-    logs: subscribeBackupNgLogs,
+    logs: cb =>
+      subscribeBackupNgLogs(logs =>
+        cb(
+          groupBy(
+            logs,
+            log => (log.message === 'restore' ? 'restore' : 'backup')
+          )
```

Comment:
Why grouping instead of simply filtering?

Reply:
because i'll display the restore logs with the backup logs.